### PR TITLE
admin/userManagement route

### PR DESCRIPTION
**Pull request by rfontanarosa**
part of #1654

Schermata 2022-10-26 alle 16 16 54

### CODE REVIEW

File: src/meta/app.ts [hunk 1]
```diff
@@ -37,13 +37,19 @@ export enum AssessmentHomeRouteNames {
   links = 'links',
 }
 
+export enum AdminRouteNames {
+  userManagement = 'userManagement',
+  dataExport = 'dataExport',
```
Comment (sorja) [inline src/meta/app.ts]: is dataExport still used?

Comment (rfontanarosa) [reply src/meta/app.ts]: idk, I can comment it for now, as the section value in Admin.tsx

File: src/client/pages/Admin/Admin.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+.app-view__content {
+  grid-area: content;
+  height: 100%;
+  padding: 28px 32px 48px 32px;
+  overflow-x: auto;
+  overflow-y: hidden;
+}
+
+.landing__page-header {
```
Comment (minotogna) [inline src/client/pages/Admin/Admin.scss]: this is not landing. this is admin 
rename the css according to the component

**Review by minotogna — CHANGES_REQUESTED** [review]

**Review by sorja — APPROVED** [review]

File: src/client/pages/Admin/Admin.scss (hunk 2 above)
Comment (minotogna) [reply src/client/pages/Admin/Admin.scss]: or check if those rule have already been defined in landing page

**Review by minotogna — APPROVED** [review]

**Pull request merged**